src/components/crossDevice/Intro/index.js

```diff
@@ -21,24 +21,27 @@ const Intro = ({translate, nextStep, mobileConfig}) => {
     ),
     'return-computer': 'return_computer',
   }
+  const stageList = Object.keys(stages)
 
   return (
     <div className={theme.fullHeightMobileContainer}>
       <PageTitle
         title={translate(`cross_device.intro.${ isFace ? 'face' : 'document' }.title`)}
       />
-      <div className={classNames(theme.thickWrapper, style.content)}>
+      <ol
+        aria-label={translate('accessibility.continue_on_mobile_steps')}
```

Comment:
We are also displaying this screen for document. Do we use "Face verification" for both document and selfie? 🤔

Reply:
You are right! Have made the change to take this into consideration.